tests/unit/test_serialize.py

```diff
@@ -440,3 +440,82 @@ def setUp(self):
     def test_always_serialized_as_str(self):
         request = self.serialize_to_request({'ContentLength': 100})
         self.assertEqual(request['headers']['Content-Length'], '100')
+
+
+class TestRestXMLUnicodeSerialization(unittest.TestCase):
+    def setUp(self):
+        self.model = {
+            'metadata': {'protocol': 'rest-xml', 'apiVersion': '2014-01-01'},
+            'documentation': '',
+            'operations': {
```

Comment:
I'd prefer to keep the test data smaller provided we're covering the same code paths from the actual s3 model.